miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/view/RequisitionController.java

```diff
@@ -135,6 +163,12 @@ private ModelAndView setupForm(Requisition requisition, PageMode pageMode, Model
     model.put(PageMode.PROPERTY, pageMode.getLabel());
     model.put("requisition", requisition);
     model.put("requisitionDto", mapper.writeValueAsString(RequisitionDto.from(requisition)));
+
+    // if creating a requisition, return empty variable used in editRequisition.jsp to prevent errors
+    if (!model.containsKey("potentialAssays")) {
+      List<Long> emptyAssayIds = new ArrayList<>();
+      model.put("potentialAssays", mapper.writeValueAsString(emptyAssayIds));
+    }
```

Comment:
Yes, all assays are available to choose from when creating an assay.

I put it as an empty list in the model because if it isn't there, then calling it in editRequisition.jsp would cause an error. If, for example, I had:
`var holder = ${potentitalAssays};`
 and if I didn't add it to the model, then it would become
` var holder = ; `
 which causes an error. 
 
 I've tried using conditionals to catch this in different ways but haven't found a way without adding a bunch of extra code.

Reply:
Only access it if the `pageMode == "edit"`